### PR TITLE
add node stats os mem stats

### PR DESCRIPTION
**Pull request by philiplhchan**

### CODE REVIEW

File: collector/nodes.go [hunk 1]
```diff
@@ -212,6 +212,54 @@ func NewNodes(logger log.Logger, client *http.Client, url *url.URL, all bool, no
 				},
 				Labels: defaultNodeLabelValues,
 			},
+			{
+				Type: prometheus.GaugeValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "os", "mem_free"),
```
Comment (zwopir) [inline collector/nodes.go]: I'd suggest to follow the convention to append the unit, so `mem_free_bytes`

File: collector/nodes.go [hunk 2]
```diff
@@ -212,6 +212,54 @@ func NewNodes(logger log.Logger, client *http.Client, url *url.URL, all bool, no
...
+				),
+				Value: func(node NodeStatsNodeResponse) float64 {
+					return float64(node.OS.Mem.Free)
+				},
+				Labels: defaultNodeLabelValues,
+			},
+			{
+				Type: prometheus.GaugeValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "os", "mem_used"),
```
Comment (zwopir) [inline collector/nodes.go]: same here and in the other mem_*_metrics

**Comment by philiplhchan** [comment]
thanks for the feedback, updated.

**Comment by zwopir** [comment]
looks good, thanks a lot for your contribution!

**Pull request merged**